src/app/components/ATIAnalytics/atiUrl/index.ts

```diff
@@ -565,6 +526,27 @@ export const buildReverbPageSectionEventModel = ({
         isSignedIn: false,
       },
     },
-    eventDetails,
+    eventDetails: {
+      eventName: type === 'view' ? 'sectionView' : 'sectionClick',
```

Comment:
Can we use this const:
```suggestion
      eventName: type === VIEW_EVENT ? 'sectionView' : 'sectionClick',
```